notes-with-indexeddb/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+
+    "description": "Examplifies working with IndexedDB in a background page",
+    "manifest_version": 2,
+    "name": "notes-with-indexeddb",
```

Comment:
This is meant to be a human-readable name, so it should be something like `"Notes with IndexedDB"`